with-realm/eas.json

```diff
@@ -0,0 +1,25 @@
+{
+  "builds": {
```

Comment:
it looks like the eas.json used to generate this is very old - i believe from august. it has changed a bunch since then. you should reinstall it and ensure you're on the latest version (0.42.4 at the time of writing), then delete `eas.json` and run `eas build:configure` (we do migrate the config but in this case i think it's better to just generate the new `eas.json` since we also added some default profiles)

Reply:
the new `eas.json` has been generated using 0.42.4